docs/examples/ukf-cholesky-vs-ldlt.md

```
@@ -0,0 +1,237 @@
+cmake --build build --target example_ukf_comparison example_bearing_only_ukf
+./build/examples/example_ukf_comparison
+./build/examples/example_bearing_only_ukf
+```
+
+## Results and Interpretation
+
+### Double precision: both methods perform identically
+
+In float64, both Cholesky and LDL^T succeed across all scenarios. The condition numbers reach ~10^4, well within the ~10^15 that double precision can handle:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)